bitcoin/src/consensus/params.rs

```diff
@@ -61,6 +71,7 @@ impl Params {
                 rule_change_activation_threshold: 1916, // 95%
                 miner_confirmation_window: 2016,
                 pow_limit: Work::MAINNET_MIN,
+                pow_limit_target: Target::from_le_bytes(Work::MAINNET_MIN.to_le_bytes()),
```

Comment:
This is wrong. You need to get the values from Bitcoin Core.

Reply:
>This is wrong. You need to get the values from Bitcoin Core.

I was surprize by the opposite 😄 as I mentioned in my previous message I do not know if the value of Targed is done correctly. I would like to discuss the solution to add it without breaking change

Reply:
Since we are uncovering bugs I vote for just doing a breaking change and getting the whole thing right.

In general we use `deprecated` when possible and give folk one version before we remove it. This is mainly to try and be gentle on users but since we are pre-1.0 and are trying _really_ hard to stabalize getting a good API takes precedence over deprecating and/or keeping kruft around.

Reply:
Make sense, I will finish the patch today. I spent some time understanding the different between the Work type and the Target type before